src/data/content/community.yaml

```diff
@@ -231,6 +231,19 @@ components:
 
       cards:
 
+      - title: "Come è cambiata la community di Designers Italia e Developers Italia?"
+        headingLevel: 4
+        text: "Ecco i primi risultati dell’indagine condotta negli scorsi mesià"
+        img: /images/community/20230711-come-e-cambiata-la-community-di-designers-italia-e-developers-italia-1400x723.png
+        alt: ""
+        imgRatio: 21x9
+        # imgPlaceholder: true
+        fullHeight: true
+        rounded: true
+        url: "/community/notizie/20230711-come-e-cambiata-la-community-di-designers-italia-e-developers-italia/"
+        dateInfo: 11 luglio 2023
+        tags: [Community, Ricerca utente]
```

Comment:
```suggestion
        tags: [Partecipazione, Ricerca utente]
```

Reply:
we've removed the tag "community"during the last "tag refactoring" iteration; we can use "partecipazione" instead